staff/bamidele-aighewi/precourse/calculator-html/index.js

```diff
@@ -0,0 +1,194 @@
+"use strict"
+
+var STRING_ACTIONS = {
+    reset: "reset",
+    backspace: "backspace",
+    division: "division", /* symbols: / ÷ */
+    multiplication: "multiplication", /* symbols: x * */
+    subtraction: "subtraction",
+    addition: "addition",
+    comma: "comma", /* symbols: . */
+    equal: "equal",
+}
+
+var inputBuffers = [];
+var uiBuffers = [];
+var currentBuffer = "";
+var result = 0;
+var equation = false;
+var messageTimeout = undefined;
```

Comment:
you can remove "= undefined;" and by default messageTimeout will be undefined. 

For instance only put: var messageTimeout;

Reply:
Touché monsieur :)